spyder_terminal/terminalplugin.py

```diff
@@ -57,11 +57,18 @@ def __init__(self, parent):
         self.tab_widget = None
         self.menu_actions = None
         self.port = select_port(default_port=8070)
+
+        stdout_file = osp.join(getcwd(), 'spyder_terminal_out.log')
+        stderr_file = osp.join(getcwd(), 'spyder_terminal_err.log')
+        self.server_stdout = open(stdout_file, 'w')
+        self.server_stderr = open(stderr_file, 'w')
+
         self.server = subprocess.Popen(
             [sys.executable, osp.join(LOCATION, 'server', 'main.py'),
              '--port', str(self.port)],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE)
+            stdout=self.server_stdout,
+            stderr=self.server_stderr)
```

Comment:
Let's make the server write to a file only if `DEV` is `True`. `DEV` can be imported from `config.base` and it's `True` when you start Spyder with `bootstrap.py`.